test/containerized_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+import json
+import http.client as http_client
+
+import pytest
+
+from ovirt_imageio._internal import config
+from ovirt_imageio._internal import server
+from ovirt_imageio._internal.units import KiB
+
+from . import testutil
+from . import http
+
+
+TICKET_IMG_UUID = "file"
+TICKET_IMG_SIZE = 16 * KiB
+
+
+@pytest.fixture(scope="module")
+def tmp_image(tmpdir_factory):
+    img_path = tmpdir_factory.getbasetemp() / "disk.raw"
+    with open(img_path, "wb") as f:
+        f.truncate(TICKET_IMG_SIZE)
+    yield img_path
+
+
+@pytest.fixture(scope="module")
+def tmp_ticket(tmpdir_factory, tmp_image):
+    ticket_path = tmpdir_factory.getbasetemp() / "file.json"
+    ticket = testutil.create_ticket(
+        uuid=TICKET_IMG_UUID, size=TICKET_IMG_SIZE, url=f'file://{tmp_image}')
+    ticket_path.write(json.dumps(ticket))
+    return str(ticket_path)
+
+
+@pytest.fixture(
+    scope="module",
+    params=["daemon", "proxy"]
```

Comment:
The proxy config is not relevant for this test, so we can test only using the daemon conf.